photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipeline.java

```diff
@@ -79,28 +79,7 @@ protected void setPipeParamsImpl() {
         // target model for the draw 3d targets pipeline to work...
 
         // for now, hard code tag width based on enum value
-        double tagWidth;
```

Comment:
We still have a family dropdown tho, don't we?

Reply:
Dropdown controls the 3d model, which in turn is just a set of coordinates. Single-side-of-tag isn't something I'm seeing anywhere... yet....

![image](https://user-images.githubusercontent.com/4583662/210152785-128d0133-b9a1-48b6-8d4d-659ea124f356.png)

This seems slightly smelly, given that not all tags have "sides" - see circular tags, or nested tags. We should probably talk to wpilib about it.

The hardcoding has been done, and the comment above explains it pretty well. Basically, longer term, if other size tags are needed, we just gotta update our data model (and figure out how we wanna, maybe, derive tag shape information from a single "sides" input)

Reply:
It says `// for now, hard code tag width based on enum value`, but it looks like this commit gets rid of the enum value switch-case that was there before

```java
        switch (settings.targetModel) {
            case k200mmAprilTag:
                {
                    tagWidth = Units.inchesToMeters(3.25 * 2);
                    break;
                }
            case k6in_16h5:
                {
                    tagWidth = Units.inchesToMeters(3 * 2);
                    break;
                }
            default:
                {
                    // guess at 200mm?? If it's zero everything breaks, but it should _never_ be zero. Unless
                    // users select the wrong model...
                    tagWidth = 0.16;
                    break;
                }
        }
```